src/ucp/core/ucp_context.c

```diff
@@ -36,6 +36,13 @@ static const char * ucp_device_type_names[] = {
     [UCT_DEVICE_TYPE_SELF] = "loopback",
 };
 
+static const char * ucp_rndv_modes[] = {
+    [UCP_RNDV_MODE_GET_ZCOPY] = "get_zcopy",
+    [UCP_RNDV_MODE_PUT_ZCOPY] = "put_zcopy",
+    [UCP_RNDV_MODE_AUTO]      = "auto",
```

Comment:
I do not see where "auto" value is handled.. Below, there are checks in if-statements to exact "put" or "get" values with 'else' cases. So, if user defines this value, behaviour may be erroneous, right? Pls, add a test for this value too.